core/src/main/resources/jenkins/security/apitoken/ApiTokenPropertyDisabledDefaultAdministrativeMonitor/message.properties

```diff
@@ -0,0 +1,2 @@
+warningMessage=A Legacy API Token is currently generated when a new user is created. <br />\
```

Comment:
nit: I would reword this to "A legacy API token will be generated automatically for newly created users.\<br/\>This behavior is supported for backwards compatibility, but legacy API tokens are deprecated and are not recommend for long-term use. Users should instead generate API tokens as needed from the user configuration page."

Reply:
correction applied with fixed typo: recommend**ed**